### PR TITLE
use segment names similar to those created by python-swiftclient

### PR DESCRIPTION
**Pull request by kahing**
Currently swift creates blob-1, blob-2... blob-n for multipart
upload segments, which are very common names that can easily
collide with normal objects. This changes the naming convention
to that of python-swiftclient (the `swift' command line).

python-swiftclient also uses a different segment container by
default, that's not implemented by this patch

### CODE REVIEW

File: apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java [hunk 1]
```diff
@@ -410,7 +410,16 @@ public long countBlobs(String containerName) {
 
    @Override
    public MultipartUpload initiateMultipartUpload(String container, BlobMetadata blobMetadata) {
-      String uploadId = UUID.randomUUID().toString();
```
Comment (andrewgaul) [inline apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java]: Call `initiateMultipartUpload(container, blobMetadata, 0)`?

**Comment by andrewgaul** [comment]
@kahing Overall agree with the approach.

@zack-shoylev What do you think?

File: apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java (hunk 1 above)
Comment (kahing) [inline apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java]: done

**Comment by andrewgaul** [comment]
Pushed to master as b8b20c16f5e7d0248e7611125b10c4ec45895910.

**Pull request closed without merging**